app/services/power_manager/bmc.rb

```diff
@@ -18,11 +18,9 @@ def action_map
       super.deep_merge({
                          :start    => 'on',
                          :stop     => 'off',
-                         :poweroff => 'off',
-                         :reboot   => 'soft',
-                         :reset    => 'cycle',
-                         :state    => 'status',
-                         :ready?   => 'ready?',
+                         :poweroff => 'soft',
+                         :reboot   => 'cycle',
+                         :state    => { :action => 'status' },
                        })
```

Comment:
I like poweroff=soft, makes sense. But reboot=cycle is very dangerous if we want to keep using the term we need to implement some kind of UI mapping for WebUI/CLI. But since this PR looks like a bigger refactoring, why not to completely rename those symbols to safe and meaningful variants?